### PR TITLE
feat: add testing env

### PR DESCRIPTION
**Pull request by cabelitos**
This commit the support for testing the file upload support in action.
The main object is to create a graphql API which accepts file as input
and returns the file content as a response. This is preferred over jest
testing, since we can't fully test connection between apollo gateway
and the micro service using jest.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -34,10 +34,11 @@ jobs:
         key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
         restore-keys: |
           ${{ runner.os }}-yarn-
+    - run: cp .env.example .env
     - run: yarn --frozen-lockfile
     - run: yarn install-peers -f
     - run: yarn lint
     - run: yarn build
-    - run: yarn test --coverage
+    - run: yarn test
```
Comment (barbieri) [inline .github/workflows/ci.yml:42]: why is that?

File: upload1.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+upload1
```
Comment (barbieri) [inline upload1.txt]: move these files to `test/` folder

File: .github/workflows/ci.yml (hunk 1 above)
Comment (cabelitos) [reply .github/workflows/ci.yml:42]: This is not done via jest, so it has no coverage.

File: upload1.txt (hunk 2 above)
Comment (cabelitos) [reply upload1.txt]: ok

**Comment by cabelitos** [comment]
@barbieri I added cod cov with instambul

File: .github/workflows/ci.yml (hunk 1 above)
Comment (cabelitos) [reply .github/workflows/ci.yml:42]: This is done by default via nyc command (instambul)

**Review by JanaMarinS — DISMISSED** [review]

**Pull request merged**